<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1246</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="canvas" name="field" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>831</width>
      <height>661</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>30</y>
      <width>211</width>
      <height>195</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Цвета</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="1">
         <widget class="color_view" name="vis_part_col" native="true"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="vis_part_col_btn">
          <property name="text">
           <string>Видимая часть</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="color_view" name="seg_col" native="true"/>
        </item>
        <item row="2" column="1">
         <widget class="color_view" name="cut_col" native="true"/>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="seg_col_btn">
          <property name="text">
           <string>Исходный отрезок</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="cut_col_btn">
          <property name="text">
           <string>Границы отсекателя</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Отсечь</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1246</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>canvas</class>
   <extends>QWidget</extends>
   <header>canvas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>color_view</class>
   <extends>QWidget</extends>
   <header>color_view.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
